### PR TITLE
add: Badge and Button Component

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added badge and button component. Ignore the commented out css code for floating-action-button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,5 +172,163 @@ <h3 class="mg-sm-top-bottom">Combining Avatar Classes</h3>
         class="avatar avatar-xlg avatar-outline-warning"
```
Comment (vrx29) [inline index.html:172]: not sure which tag is this. Looks like a typo.

File: Components/badge.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/* BADGE COMPONENT */
+
+/* Badge on Avatar */
+
+.badge-on-avatar {
```
Comment (arung86) [inline Components/badge.css:20]: suggest you move these lines below, as you don't have badge in the top and directly talking about the avatar and badge

File: Components/badge.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  width: max-content;
+  padding: 1rem;
+}
+
+.badge-on-icon i {
+  font-size: 1.5rem;
+}
+
+.badge {
+  font-family: inherit;
```
Comment (arung86) [inline Components/badge.css]: move this to global declarations / common / resets

File: Components/button.css [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  padding: 0.5rem 1rem;
+  margin: 0.25rem;
+  background: transparent;
+  text-align: center;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  font-size: 0.8rem;
+  cursor: pointer;
+  transition: background 0.15s;
+  font-family: inherit;
```
Comment (arung86) [inline Components/button.css]: font-family is an inherited property, so this line doesn't make any sense

Comment (ParthParolekar) [reply Components/button.css]: Fixing it right now.

File: Components/badge.css (hunk 2 above)
Comment (ParthParolekar) [reply Components/badge.css:20]: Oh, right! Makes sense.

File: index.html (hunk 1 above)
Comment (ParthParolekar) [reply index.html:172]: I didn't understand your review properly.  I don't see any typo. Can you elaborate?

File: Components/badge.css (hunk 3 above)
Comment (ParthParolekar) [reply Components/badge.css]: I took this line out. Just like in the button component, font-family is already inherited so no need for this. thanks for the review

**Pull request merged**